docs/tutorials/aws.md

```diff
@@ -80,21 +93,60 @@ spec:
     spec:
       containers:
       - name: external-dns
-        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.2
+        image: registry.opensource.zalan.do/teapot/external-dns:v0.4.7
         args:
         - --source=service
         - --source=ingress
         - --domain-filter=external-dns-test.my-org.com # will make ExternalDNS see only the hosted zones matching provided domain, omit to process all available hosted zones
         - --provider=aws
         - --policy=upsert-only # would prevent ExternalDNS from deleting any records, omit to enable full synchronization
+        - --aws-zone-type=public #only look at public hosted zones (valid values are public/private)
         - --registry=txt
         - --txt-owner-id=my-identifier
 ```
 
-## Verify ExternalDNS works
+## Arguments
+
+This list is not the full list, but a few arguments that where choosen.
+
+### aws-zone-type
+
+`aws-zone-type` allows filtering for private and public zones
+
+
+## Verify ExternalDNS works (Ingress example)
+
+Change your ingress resource manifest file. In example below, I am using nginx ingress controller
+
+> For ingress objects ExternalDNS will create a DNS record based on the host specified for the ingress object.
+
+
+```yaml
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: foo
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/ssl-redirect: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
```

Comment:
Let's leave out the annotations here to keep it focused on AWS. These annotations aren't necessary to understand ExternalDNS at this point. There's a separate tutorial file for ExternalDNS + nginx controller in this folder where we could explain them in more detail.